test/suite/unit_test_suite.go

```diff
@@ -2,10 +2,8 @@ package suite
 
 import (
```

Comment:
what about removing this file in favor of https://github.com/fabric8-services/fabric8-cluster/tree/master/test/suite ?